modules/kubernetes-addons/variables.tf

```diff
@@ -161,6 +161,24 @@ variable "amazon_prometheus_workspace_endpoint" {
   description = "AWS Managed Prometheus WorkSpace Endpoint"
 }
 
+variable "amazon_prometheus_workspace_region" {
+  type        = string
+  default     = null
+  description = "AWS Managed Prometheus WorkSpace Region"
+}
+
+variable "amazon_prometheus_ingest_iam_role_arn" {
```

Comment:
Seems like you are actually creating this role in the module so no need to pass this.